tests/Jira/Api/Authentication/BasicTest.php

```diff
@@ -1,13 +1,17 @@
 <?php
 
-class Jira_Api_Authentication_BasicTest extends PHPUnit_Framework_TestCase
+namespace Tests\chobie\Jira;
+
+use chobie\Jira\Api\Authentication\Basic;
+
+class Jira_Api_Authentication_BasicTest extends \PHPUnit_Framework_TestCase
```

Comment:
Since you namespace test classes why not rename `Jira_Api_Authentication_BasicTest` into `BasicTest` to make autoloading work?